crates/swc_ecma_transforms_base/src/hygiene/usage_analyzer.rs

```diff
@@ -580,8 +580,14 @@ impl Visit for UsageAnalyzer<'_> {
     fn visit_member_expr(&mut self, e: &MemberExpr) {
         e.obj.visit_with(self);
 
-        if e.computed {
-            e.obj.visit_with(self);
```

Comment:
I suppose this should be `e.prop.visit_with` @kdy1 ?

Reply:
Yes you are right

Reply:
Oh, let's spin a separate pr real quick to avoid massive snapshot overwrite...